src/Native/System.Native/pal_networkstatistics.cpp

```diff
@@ -412,4 +413,113 @@ extern "C" int32_t GetActiveUdpListeners(IPEndPointInfo* infos, int32_t* infoCou
     return 0;
 }
 
+extern "C" int32_t GetNativeIPInterfaceStatistics(char* interfaceName, NativeIPInterfaceStatistics* retStats)
```

Comment:
Is this called from a passthrough from consumers or is this called after processing from the libraries? If this isn't used as a passthrough, we should assert that the pointers are not nullptr